test/e2e/federated-service.go

```diff
@@ -344,10 +349,36 @@ func discoverService(f *framework.Framework, name string, exists bool) {
 
 	if exists {
 		// TODO(mml): Eventually check the IP address is correct, too.
-		Eventually(podExitCodeDetector(f, FederatedServicePod, 0), DNSTTL, time.Second*2).
+		Eventually(podExitCodeDetector(f, FederatedServicePod, 0), 10*DNSTTL, time.Second*2).
 			Should(BeNil(), "%q should exit 0, but it never did", command)
 	} else {
-		Consistently(podExitCodeDetector(f, FederatedServicePod, 0), DNSTTL, time.Second*2).
+		Consistently(podExitCodeDetector(f, FederatedServicePod, 0), 10*DNSTTL, time.Second*2).
 			ShouldNot(BeNil(), "%q should never exit 0, but it did", command)
 	}
 }
+
+func createBackendPods(clusterClientSets []*release_1_3.Clientset, namespace string) {
+	name := "backend"
+
+	pod := &v1.Pod{
```

Comment:
nit: Probably more resilient if you create a replcaset of 1 here.  That way if it crashes or ends up on a dead node or similar, a new one will be created automatically.